src/engine_core/rendering/src/Vulkan/VkUtilsFactory.hpp

```diff
@@ -283,6 +283,21 @@ class VkUtilsFactory
 		info.pPushConstantRanges = nullptr;
 		return info;
     }
+	static VkRenderingAttachmentInfo DepthAttachmentInfo(
+		VkImageView view, VkImageLayout layout /*= VK_IMAGE_LAYOUT_COLOR_ATTACHMENT_OPTIMAL*/)
```

Comment:
just one question, why is the `= VK_IMAGE_LAYOUT_COLOR_ATTACHMENT_OPTIMAL` commented?

Reply:
The editor added this automatically for the optional parameter, we can remove it tho c: